src/ray/protobuf/core_worker.proto

```diff
@@ -158,35 +156,37 @@ message WaitForObjectEvictionRequest {
   bytes object_id = 2;
 }
 
-message WaitForObjectEvictionReply {
-}
+message WaitForObjectEvictionReply {}
 
 message AddObjectLocationOwnerRequest {
   bytes intended_worker_id = 1;
   bytes object_id = 2;
   bytes node_id = 3;
 }
 
-message AddObjectLocationOwnerReply {
-}
+message AddObjectLocationOwnerReply {}
 
 message RemoveObjectLocationOwnerRequest {
   bytes intended_worker_id = 1;
   bytes object_id = 2;
   bytes node_id = 3;
 }
 
-message RemoveObjectLocationOwnerReply {
-}
+message RemoveObjectLocationOwnerReply {}
 
 message GetObjectLocationsOwnerRequest {
   bytes intended_worker_id = 1;
   bytes object_id = 2;
+  // The version of the last location update. Only updates more recent than this version
+  // will be returned.
+  uint64 last_version = 3;
```

Comment:
```suggestion
  int64 last_version = 3 [default=-1];
```

Avoid using unsigned types due to overflow issues

Reply:
I was planning to use a `uint64` and have 0 be the sentinel given that
1. that's the default proto value for numeric types,
2. explicit default values aren't allowed in proto,
but I agree that the semantics of unsigned overflow are not great. I'll make this a signed int64, but we won't have a proto-level default.